### PR TITLE
Dan 285

### PR DESCRIPTION
**Pull request by ErikVWT**

### CODE REVIEW

File: dashboard/data/data.py [hunk 1]
```diff
@@ -24,3 +25,60 @@ def has_planning_by_month():
     end_date = pd.to_datetime(str(datetime.now().year) + '-12-31', format="%Y-%m-%d")
     mask = (df['date'] >= start_date) & (df['date'] <= end_date)
     return df[mask]
+
+
+def completed_status_counts(project_name, click_filter=None):
+    StatusCountDataFrames = namedtuple("StatusCountDataFrames",
```
Comment (avanturnhout) [inline dashboard/data/data.py:31]: add logging

File: dashboard/layout/pages/tmobile/detail.py [hunk 2]
```diff
@@ -1,5 +1,20 @@
 import dash_html_components as html
 
+from data.data import completed_status_counts
+from layout.components.graphs import completed_status_counts_bar
```
Comment (avanturnhout) [inline dashboard/layout/pages/tmobile/detail.py:4]: detail veranderen in project?

Comment (ErikVWT) [reply dashboard/layout/pages/tmobile/detail.py]: Done

File: dashboard/data/data.py (hunk 1 above)
Comment (ErikVWT) [reply dashboard/data/data.py:31]: Done in FttX.py:152

**Review by avanturnhout — APPROVED** [review]

**Pull request merged**